library-backend/library-api/pom.xml

```diff
@@ -9,6 +9,32 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <artifactId>library-api</artifactId>
+    <dependencies>
```

Comment:
Let's create a new maven module under the root `pom.xml` called `spring-learning` where you'll add this so we don't pollute the existing modules and their functionalities.

As we've discussed we should never add dependencies on frameworks in `api` modules.